### PR TITLE
updated the readme file by adding a new step in Development section

### PR DESCRIPTION
**Pull request by mrkarimoff**
Updated the readme file by adding a new step of running `npm run preelectron:dev` in Development section and fixed the required NPM version to 8.3.2 to match the package.json file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,20 +45,20 @@ Note: for Apple Silicon users, you need to install the `electron` package manual
...
+| `test`                          | Runs testing suite                                                                                                             |
+| `preflight`                     | Runs linting & testing. Useful as a prepush/build hook                                                                         |
+| `dist:mac`                      | Build and publish OS X verison                                                                                                 |
+| `dist:linux`                    | Build and publish Linux version                                                                                                |
+| `dist:win`                      | Build and publish Windows version                                                                                              |
+| `dist:all`                      | Build and publish all platforms                                                                                                |
+| `update-submodules`             | Update git submodules                                                                                                          |
```
Comment (jthrilly) [inline README.md:61]: Let's add the `preelectron:dev` command here, for reference.

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ For questions and support, please visit the [Network Canvas User Community](http
 ### Prerequisites
 
 - [Node.js](https://nodejs.org/en/) (v14.21.3)
-- [npm](https://www.npmjs.com/) (v8.19.4)
+- [npm](https://www.npmjs.com/) (v8.3.2)
```
Comment (jthrilly) [inline README.md:16]: I think this was correct previously - if package.json says 8.3.2 I would suggest updating that, instead.

File: README.md [hunk 3]
```diff
@@ -75,13 +75,18 @@ e.g.
...
+
 1. `npm run start:network-canvas:electron`: to start the webpack dev server
-  - Note: must be running on port 3000.
-2. `npm run electron:dev` (in another terminal session)
-  1. Copies the electron source to `./electron-dev`
-  2. Runs the electron app from there
 
+- Note: must be running on port 3000.
+
+2. `npm run preelectron:dev` (in another terminal session)
```
Comment (jthrilly) [inline README.md:85]: I think we should make it clear that this step only needs to happen when you first set up the repo, or when you bump the version number.

**Review by jthrilly — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mrkarimoff) [reply README.md:16]: Sure, will do sir

**Review by jthrilly — APPROVED** [review]

**Pull request merged**